devvit.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://developers.reddit.com/schema/config-file.v1.json",
+  "name": "YOUR_APP_NAME",
+  "post": {
+    "client": {
+      "dir": "src/client/dist",
+      "entry": "src/client/dist/index.html"
+    },
+    "create": {
+      "height": "tall",
+      "menu": {
+        "enable": true,
+        "label": "Hello World",
+        "scope": "moderator"
+      }
+    }
+  },
+  "server": {
+    "entry": "src/server/dist/index.js"
```

Comment:
consider moving to dist to the root. outputing under src/ is unusual except for codegen in my opinion.